app/pdfs/certificate_pdf.rb

```diff
@@ -0,0 +1,48 @@
+require 'open-uri'
+
+class CertificatePDF < Prawn::Document
+  attr_accessor :path
+
+  PDF_OPTIONS = {
+    page_size:   'A5',
+    page_layout: :landscape,
+    margin:      [30, 75]
+  }
+
+  def initialize(certificate)
+    super(PDF_OPTIONS)
+
+    image open(certificate.cover_url), width: 450, align: :center
+
+    move_up 40
+    fill_color 'ffffff'
+    text I18n.t('.certificates.pdf.title'), size: 20, align: :center
+
+    fill_color '40464e'
+
+    move_down 40
+    text I18n.t('.certificates.pdf.description',
+                participant_name: certificate.participant_name,
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.